lms/static/sass/course/modules/_student-notes.scss

```diff
@@ -4,17 +4,50 @@
 // in this document:
```

Comment:
NOTE: this is a module-based Sass partial for non view-specific Student Notes styling (for now, that means UI in creating/viewing/editing notes in Courseware).